.github/PULL_REQUEST_TEMPLATE/dev.md

```diff
@@ -0,0 +1,47 @@
+## Connected Issues
```

Comment:
What would the experience of multiple PR templates look like on Github? (Try it out on a sample project and add screenshots if possible)

Reply:
The template for PRs adding a new blog will be the default template. To change to the `dev.md` template, when creating the PR, users will have to add a `template=dev.md` query param. I've added this explanation to the `CONTRIBUTING.md` file.